src/Common/src/System/Buffers/ArrayBufferWriter.cs

```diff
@@ -9,25 +9,23 @@ namespace System.Buffers
     /// <summary>
     /// Represents a heap-based, array-backed output sink into which <typeparam name="T"/> data can be written.
     /// </summary>
-#if USE_ABW_INTERNALLY
-    internal
-#else
+#if MAKE_ABW_PUBLIC
```

Comment:
Condition inverted. @joperezr, please review the csproj changes related to this.

Reply:
**Note** however, with this condition inversion, the source package doesn't mark the type internal for both if/defs. It isn't a concern, but just bringing it up as a "diff":

**Before:**
```C#
#if USE_ABW_INTERNALLY
    internal
#else
    internal
#endif
    sealed class ArrayBufferWriter<T> : IBufferWriter<T>
```

**Now:**
```C#
#if MAKE_ABW_PUBLIC
    public
#else
    internal
#endif
    sealed class ArrayBufferWriter<T> : IBufferWriter<T>
```

Reply:
This should not (will not) have any risk in terms of source package consumption. But regardless, letting you know, just in case. :)
cc @BrennanConroy